tests/src/test/scala/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -269,4 +269,29 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers {
             }
     }
 
+    it should "limit the length of HTTP req/resp bodies for debug messages to 1000 bytes" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+            val name = "largeFileAction"
+            val debugMsg = "Body excedes 1000 bytes and will be truncated"
```

Comment:
From my understanding, maybe @mdeuser can elaborate more on this, we currently print out the req/resp body a total of 4 times (2 of those being debugs) during a `wsk action create xxx -d`. If you take a large file, the amount of output can overwhelm the CLI and even hang it.  Maybe this fixes something that is too rare to matter if there are more frequent disadvantages?